src/common/options/global.yaml.in

```diff
@@ -6718,3 +6718,10 @@ options:
   desc: Enables exception throwing instead of process abort on transaction submission error.
   default: false
   with_legacy: false
+- name: ceph_assert_supresssions
+  type: str
+  desc: Suppress specific ceph_assert instances to let the execution continue with unknown, potentially DESTRUCTIVE consequences.
+  fmt_desc: Comma-separated list of locations of ceph_assert in the source code.
+    The format is ``{file}:{line} [, {file}:{line}]``
```

Comment:
is it the absolute path, relative path, or just the .cc/.h filename that's needed?

Reply:
I don't recall for sure, to be honest. I think it's full absolute with the file extension.
Though, what I'm sure about is it's entirely consistent with the assert message printed by the assert handler:

```cpp
    snprintf(g_assert_msg, sizeof(g_assert_msg),
	     "%s: In function '%s' thread %llx time %s\n"
	     "%s: %d: FAILED ceph_assert(%s)\n",
	     file, func, (unsigned long long)pthread_self(), tss.str().c_str(),
	     file, line, assertion);
    dout_emergency(g_assert_msg);
```

Reply:
it'd be good to clarify in the documentation

Reply:
I had a call with @jdurgin today. As the audience are developers, mostly working on distaster recovery, there is no point in making this too user-friendly.